.github/actions/javascript/proposalPoliceComment/action.yml

```
@@ -0,0 +1,15 @@
+name: 'ProposalPolice™ Comment Check Script'
+description: 'Verifies whether a proposal comment follows the template OR an updated proposal had substantial changes'
+inputs:
+  GITHUB_TOKEN:
+    description: 'Auth token for New Expensify Github'
+    required: true
+  OPENAI_API_KEY:
+    description: 'OpenAI API key for accessing AI services'
+    required: true
+  OPENAI_ASSISTANT_ID:
```

And this one to `PROPOSAL_POLICE_ASSISTANT_ID`

```suggestion
  PROPOSAL_POLICE_ASSISTANT_ID:
```